src/clients/SpokePoolClient.ts

```diff
@@ -438,14 +438,7 @@ export class SpokePoolClient extends BaseAbstractClient {
     low: number;
     high: number;
   }> {
-    return getBlockRangeForDepositId(
-      targetDepositId,
-      initLow,
-      initHigh,
-      maxSearches,
-      this.spokePool as SpokePool,
-      this.deploymentBlock
-    );
+    return getBlockRangeForDepositId(targetDepositId, initLow, initHigh, maxSearches, this, this.deploymentBlock);
```

Comment:
Passing the spoke client directly saves us an RPC call to the latest block number. It also allows us to leverage the mocked clients

Reply:
If we pass in the SpokePoolClient, then we could also drop the subsequent block number (`this.deploymentBlock`). wdyt?

Reply:
Done